packages/client/wallets/aa/src/blockchain/wallets/EVMAAWallet.ts

```diff
@@ -171,7 +167,7 @@ export class EVMAAWallet extends LoggerWrapper {
                             abi: erc20,
                             functionName: "transfer",
                             args: [toAddress, (config as ERC20TransferType).amount],
-                            ...this.getLegacyTransactionFeesParamsIfApply(),
+                            ...(usesGelatoBundler(this.chain) && gelatoBundlerProperties),
```

Comment:
The repetition I introduce here is cleaned up in https://github.com/Crossmint/crossmint-sdk/pull/502